src/main/scala/io/iohk/ethereum/Mantis.scala

```diff
@@ -11,6 +14,8 @@ object Mantis extends Logger {
     val node =
       if (Config.testmode) {
         log.info("Starting Mantis in test mode")
+        log.info(s"Deleting previous database ${Config.Db.RocksDb.path}")
```

Comment:
```suggestion
        log.info("Deleting previous database {}", Config.Db.RocksDb.path)
```
the decision to use this format uniformly ^ was made a few months ago